### PR TITLE
Add Python 3.6, drop Python 3.3, 100% coverage  

### LINKED ISSUE #3: Add Python 3.6, Drop Python 3.3

### LINKED ISSUE #5: 100% coverage

### PR DESCRIPTION
**Pull request by jamadden**
Fixes #3 and fixes #5 

Two separate commits again.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -49,25 +55,24 @@ def read(*rnames):
...
           'Programming Language :: Python :: Implementation :: CPython',
           'Programming Language :: Python :: Implementation :: PyPy',
           'Natural Language :: English',
           'Operating System :: OS Independent',
           'Topic :: Internet :: WWW/HTTP',
-          'Framework :: Zope3'],
-      url='http://pypi.python.org/pypi/zope.viewlet',
+          'Framework :: Zope3',
+      ],
+      url='http://github.com/zopefoundation/zope.viewlet',
```
Comment (mgedmin) [inline setup.py]: https:// would make me happier

File: src/zope/viewlet/metaconfigure.py [hunk 2]
```diff
@@ -113,9 +113,7 @@ def viewletDirective(
                 "Attribute and template cannot be used together.")
 
         # Note: The previous logic forbids this condition to evere occur.
```
Comment (mgedmin) [inline src/zope/viewlet/metaconfigure.py]: Typo: evere -> ever.  Actually, the whole clause should probably be rewritten to something like "forbids this condition from ever occurring", I think, but IANANS (I Am Not A Native English Speaker).

File: tox.ini [hunk 3]
```diff
@@ -1,23 +1,20 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy,pypy3
+   py27,py34,py35,py36,pypy,pypy3,coverage
```
Comment (mgedmin) [inline tox.ini]: :)

**Review by mgedmin — APPROVED** [review]
LGTM

File: src/zope/viewlet/metaconfigure.py (hunk 2 above)
Comment (jamadden) [reply src/zope/viewlet/metaconfigure.py]: Yup, typo (not mine!), and it could be phrased better.

File: tox.ini (hunk 3 above)
Comment (jamadden) [reply tox.ini]: Sorry!

File: src/zope/viewlet/metaconfigure.py (hunk 2 above)
Comment (mgedmin) [reply src/zope/viewlet/metaconfigure.py]: Sorry, didn't mean to throw any blame your way!  Just noticed an opportunity for improvement near one of the actual changes.

**Comment by jamadden** [comment]
Thank you! Could we please get a PyPI release when convenient?

**Comment by mgedmin** [comment]
You have PyPI rights now.

Also, can you do me a favour and validate the ReST syntax for errors of the setup.py --long-description?  The current PyPI page shows there was something wrong with the last release.

**Comment by jamadden** [comment]
> You have PyPI rights now.

Thanks.

> Also, can you do me a favour and validate the ReST syntax for errors of the setup.py --long-description? The current PyPI page shows there was something wrong with the last release.

I have. `restview --long --pypi` doesn't report any errors anymore. I fixed it with this PR. Previously it reported:
```
<string>:1633: (SEVERE/4) Title level inconsistent:

4.0.0 (2014-12-24)
------------------
```

**Comment by mgedmin** [comment]
Thank you!

**Pull request merged**